tests/integration/Owncloud/OcisPhpSdk/OcisResourceTest.php

```diff
@@ -284,4 +285,30 @@ public function testUploadFileStream(): void
         $content = $this->getContentOfResource425Save($resources[0]);
         $this->assertEquals('some content', $content);
     }
+
+    public function testGetRoles(): void
```

Comment:
does file and folder has same resources

Reply:
Not necessary, but that is ocis behavior, we don't need to test that, we have to work with whatever ocis sends us